boards/apollo3/lora_things_plus/src/main.rs

```diff
@@ -149,6 +148,8 @@ type SignatureVerifyInMemoryKeys =
         64,
     >;
 
+type SchedulerObj = components::sched::round_robin::RoundRobinComponentType;
```

Comment:
Bike-shedding the name for this:

* It's not an object. Arguably Rust doesn't have objects _at all_, and this is certainly is not one.
* `SchedulerType`?

Reply:
Obviously, this would apply to all the boards and is hopefully just `sed -i 's/type SchedulerObj/type SchedulerType/' boards/**/main.rs`